.travis.yml

```diff
@@ -3,6 +3,7 @@ python: "3.4"
 sudo: required
 dist: trusty
 install:
+  - if test -z "$(git diff --name-only $TRAVIS_COMMIT_RANGE -- client/client-multi/client-c)"; then exit 0; fi
```

Comment:
1. Don't use exit: https://docs.travis-ci.com/user/customizing-the-build/#How-does-this-work%3F-(Or%2C-why-you-should-not-use-exit-in-build-steps)
2. Use `${TRAVIS_COMMIT_RANGE/.../..}`. `$TRAVIS_COMMIT_RANGE` includes one extra commit (see the ticket description).